drivers/adc/Kconfig.mcux

```diff
@@ -53,6 +53,35 @@ config ADC_MCUX_ADC16_VREF_ALTERNATE
 
 endchoice
 
+config ADC16_ENABLE_EDMA
+	bool "ENABLE EDMA for adc driver"
+	depends on HAS_MCUX_ADC16 && HAS_MCUX_EDMA
+	help
+	  Enable the MCUX ADC16 driver.
+
+if ADC16_ENABLE_EDMA
+
+config ADC16_INIT_PRIORITY
+	int "Init priority"
+	default 70
+	help
+	  Device driver initialization priority.
+
+config ADC_EDMA_CHANNEL
+	int "ADC DMA Channel select"
+	depends on HAS_MCUX_EDMA
+	default 0
+	help
+	  ADC DMA Channel select (0-4) PIT triggerred
```

Comment:
I have read through the K64 reference manual a couple of times now. I cannot understand how this setting is not ADC instance specific?

Reply:
the EDMA MUX channel is ADC specific wich you can find in 3.3.9.1 DMA MUX request sources, see below
![image](https://user-images.githubusercontent.com/800198/101574334-a27fb480-3a13-11eb-8141-02d2c8556ccb.png)
so it is defined in device tree as below
dmas = <&edma0 0 40>;
as to the DMA channel it can be mapped to any channel
![image](https://user-images.githubusercontent.com/800198/101575642-01ddc480-3a14-11eb-962c-89fa327345de.png)

but to enable period trigger, only 0-4 channel is supported see 21.1.3 Modes of operation
![image](https://user-images.githubusercontent.com/800198/101576251-2df94580-3a14-11eb-9ead-fd939d198b7d.png)

Reply:
@henrikbrixandersen now the dma_channel is set to adc instance binding in dts